fac29/P1DAAF
# Diana POST endpoint

### This is a handler which will add data ( an object ) to the questionsArray -> then write this on to the data.json file.

how to test:
1. use bruno or postman (or any API client)
2. make a POST request with this `URL: http://localhost:3001/create-question`

here is a test JSON to add to the body of the request:
`{
  "id": "704",
  "category": "History",
  "difficulty": "easy",
  "question": "When was Princess Diana Born?",
  "options": ["01/07/1998", "07/07/1998", "02/02/1961", "09/09/1990"],
  "answer": "07/07/1998",
  "favourited": true,
  "timestamp": "10/10/10"
}
`


you should get a response like this
<img width="1293" alt="Screenshot 2024-05-22 at 12 47 35" src="https://github.com/fac29/P1DAAF/assets/112947016/bdcf2eeb-c347-4cbc-a312-97f664a7cb3b">



3. if you look locally on your data.json file, it should be added to the end of the file.
